.github/workflows/linters.yml.txt

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] Kindly, remove the extension `.txt` from the end of the file name, because that extension blocks the linters to run on GitHub;

Reply:
I am sorry I thought I deleted it, guess I better create the folder in the text editor than in the main folder from now on, thank you for your review